src/svg/spaceman.svg

```diff
@@ -0,0 +1,14 @@
+<svg width="186" height="180" viewBox="0 0 186 180" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Nice addition, but I don't see where this svg is being used at the moment.
What do you think about add it later, when we will use it, so that it will be clear where it is used in the app?

Reply:
It's going to be needed for the `Login Page`

Reply:
At the time of the PR, is not used anywhere, but since it's already here it doesn't hurt I guess :)